match/lib/match/generator.rb

```diff
@@ -32,8 +32,8 @@ def self.generate_certificate(params, cert_type, working_directory, specific_cer
         username: params[:username],
         team_id: params[:team_id],
         team_name: params[:team_name],
-        keychain_path: FastlaneCore::Helper.keychain_path(params[:keychain_name]),
-        keychain_password: params[:keychain_password],
+        keychain_path: Helper.mac? ? FastlaneCore::Helper.keychain_path(params[:keychain_name]) : "",
```

Comment:
It's confusing to use an empty string as a path value. Could you use `nil` instead.